core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -3573,7 +3710,7 @@ def get_base_model_names(self, model) -> list:
         base_model_set = list(self.model_graph.predecessors(model))
         return base_model_set
 
-    def model_refit_map(self, inverse=False) -> Dict[str, str]:
+    def model_refit_map(self, inverse=False) -> dict[str, str]:
```

Comment:
Could you add a TODO next to `AbstractTimeSeriesTrainer.model_refit_map` in the `__init__` method that we should reconcile the method / object?